binder/environment.yml

```diff
@@ -1,33 +1,40 @@
+# minimal environment to install hydromt and run examples
 name: hydromt
 
 channels:
   - conda-forge
 
 dependencies:
   - affine
-  - click
   - black
   - bottleneck
   - cartopy # to run examples
+  - click
   - dask
   - descartes # to run examples
   - entrypoints
   - flit>=3.2
```

Comment:
Shall we do flit 3.4 to be sure pip install works ?

Reply:
Good one. I've changed this in all environment files and the pyproj.toml